app/controllers/api/account.php

```diff
@@ -1634,6 +1634,7 @@
         }
 
         $user
+            ->setAttribute('password', $isAnonymousUser ? Auth::passwordHash($password, Auth::DEFAULT_ALGO, Auth::DEFAULT_ALGO_OPTIONS) : $user->getAttribute('password', ''))
```

Comment:
Please also make sure to update the `hash` and `hashOptions` attributes.

Reply:
@stnguyen90 Done, thanks for the review!